app/tests/mock_database.py

```diff
@@ -0,0 +1,141 @@
+import uuid
+
+from app.main.db.models import Body, Consignment, File, FileMetadata, Series, db
+
+
+def generate_mock_data():
```

Comment:
Could rename this function to something more specific like `create_two_files_with_test_string_in_fields` so that a reader looking at the test can quickly understand why the query without "test" in it, would result in 0 results, but one with "test" in it would result in 2!

Reply:
the function is creating mock data,  when called in test_search_view_with_no_results test, we are entering query search value which is specifically should not return a value, so I don't think we should change function name here , unless @colinbowen if you have any different opinion

Reply:
It's not creating mocks, it is creating actual database objects, but we only use it in our tests and so is only ever used with the test db we spin up.

And my rename suggestion was because a user can't easily verify that:
- `form_data = {"query": "junk"}` should return 0 objects,
-  `form_data = {"query": "test"}` should return 2 objects,

without having to go and read the function `generate_mock_data`, whereas if this was named `create_two_files_with_test_string_in_fields` which is a much more explicit description of the function, enough for a user not to have to go and read the whole function to understand if the test is doing the right thing or not

Reply:
renamed the function create_two_test_records